rfcs/128-community-organization.md

```diff
@@ -0,0 +1,67 @@
+- Author: Xuanwo <github@xuanwo.io>
+- Start Date: 2021-06-29
+- RFC PR: [beyondstorage/specs#128](https://github.com/beyondstorage/specs/pull/128)
+- Tracking Issue: [beyondstorage/go-storage#0](https://github.com/beyondstorage/go-storage/issues/0)
+
+# GSP-128: Community Organization
+
+## Background
+
+BeyondStorage has been grown up, and we need to define our community organization to make it clear that everyone's rights and responsibilities.
+
+## Proposal
+
+I propose to add five positions for BeyondStorage, and they are separate in different projects.
+
+For now, there are two main projects:
+
+- `go-storage`: Focus on implement storage abstraction for golang, including `go-storage`, `go-service-*`, `go-endpoint`, `site`, `specs` and so on.
```

Comment:
Is `specs` under "go-storage" project? If so, I think we should state it more clearly in the repo. Btw, there's `rust` and `definitions` seems to be general.

Reply:
Nice catch, let's split `specs` and `site` outside from `go-storage`.